docs/management/catalogdevelopment.md

```diff
@@ -0,0 +1,684 @@
+
+## Catalog components
+
+### Catalog
+
+* The top-level container for your questionnaire  
+* Contains multiple [*Sections*](#Sections)  
+* Once a catalogue has been completed it can be exported in a variety of formats.
+
+### URI prefix
+
+* A *URI Prefix* acts as a unique identifier for your RDMO instance, this is particularly important when different instances are sharing data.  
+* The default *URI Prefix* can be set in the RDMO configuration file (local.py), which usually requires administrative access to change, so if it's defined, don't change it.
+
+### URI path
+
+* It is the part of the URI that comes after the prefix.  
+* It is typically a string of characters separated by forward slashes ("/") and is primarily used as an internal identifier to locate specific items or resources within the RDMO system. When combined with the [*URI Prefix*](#URI-prefix), it forms the full URI of an item.
+
+### Sections
+
+* Main segments ofthe [*Catalog*](#Catalog).  
+* Each *Section* contains multiple [*Pages*](#Pages).  
+* *Sections* help to break the catalog down into thematic areas.
+
+### Pages
+
+* Subdivisions of *Sections* that contain and organise *Questions* and [*Question sets*](#Question-sets)  
+* A *Page* corresponds to a visible screen in the user interview process 
+
+### Collection
+
+* A collection in RDMO allows for multiple sets of answers to the same group of [*Questions*](#Questions). You can select the collection option for a [*Page*](#Pages), [*Question*](#Questions), or [*Question set*](#Question-sets). Each instance of a collection contains the same set of [*Questions*](#Questions) but can have different answers, making it useful for cases where similar information needs to be entered multiple times. This feature creates more dynamic and flexible questionnaires that adapt to different project structures. For example, if you have a [*Page*](#Pages) about "Dataset Description" and mark it as a collection, researchers can describe multiple datasets within the same project. This is particularly useful when collecting information about multiple datasets, research methods, or project partners in a single project.  
+* At the [*Question*](#Questions) level, a collection allows the answer field to a [*Question*](#Questions) to be answered more than once.  
+* At the level of [*Question sets*](#Question-sets), the collection allows [*Option sets*](#Option-sets) to be duplicated and used multiple times.  
+* At the [*Page*](#Pages) level, collections allow users to create sets (data sets or subsets of the project data, which are always represented by tabs). This means that every [*Question*](#Questions) belonging to this page must be answered for each data set.
+
+### Question Sets {#question-sets}
```

Comment:
I think that this syntax for anchor links is correct but its currently not working in the build (needs #60 )